### PR TITLE
Add object storage help

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/integrator/objectstorage.rst [hunk 1]
```diff
@@ -0,0 +1,109 @@
+.. _integrator_objectstorage:
+
+Working with Object storage (like S3)
+=====================================
+
+Generalities
+------------
+
+In this we will consider to use the S3 like from Exoscale.
```
Comment (jwkaltz) [inline doc/integrator/objectstorage.rst]: "like" is not clear. Do we mean: "to use the S3-like service from Exoscale"?

File: doc/integrator/objectstorage.rst [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+First of all we should set the following variable on the ``geoportal``, ``config`` and ``qgisserver`` services.
```
Comment (jwkaltz) [inline doc/integrator/objectstorage.rst]: variables

File: doc/integrator/objectstorage.rst [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+* ``AWS_ACCESS_KEY_ID``: The project access key.
+* ``AWS_SECRET_ACCESS_KEY``: The project secret key.
+* ``AWS_DEFAULT_REGION=ch-dk-2``: The region use by Exoscale.
```
Comment (jwkaltz) [inline doc/integrator/objectstorage.rst]: used

File: doc/integrator/objectstorage.rst [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+* ``AWS_S3_ENDPOINT=sos-ch-dk-2.exo.io``: The endpoint used by Exoscale.
+
+And for performances we should set the following variable on the ``geoportal`` and ``qgisserver`` services.
```
Comment (jwkaltz) [inline doc/integrator/objectstorage.rst]: "And for performances we should..." -> "For better performance, we should..."
"variable" -> "variables"

File: doc/integrator/objectstorage.rst [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+   CONFIG "CPL_VSIL_CURL_CACHE_SIZE" "128000000"
+   CONFIG "CPL_VSIL_CURL_USE_HEAD" "FALSE"
+   CONFIG "GDAL_DISABLE_READDIR_ON_OPEN" "TRUE"
+
+   CONFIG "AWS_ACCESS_KEY_ID" "${AWS_ACCESS_KEY_ID}"
+   CONFIG "AWS_SECRET_ACCESS_KEY" "${AWS_SECRET_ACCESS_KEY}"
+   CONFIG "AWS_DEFAULT_REGION" "${AWS_DEFAULT_REGION}"
+   CONFIG "AWS_S3_ENDPOINT" "${AWS_S3_ENDPOINT}"
+
+Use the vrt in the layer
```
Comment (jwkaltz) [inline doc/integrator/objectstorage.rst]: "layer" -> "layer:"

File: doc/integrator/objectstorage.rst [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+   CONNECTIONTYPE OGR
+   CONNECTION "/vsis3/<bucket>/<path>/<name>.shp"
+   DATA "<name>"
+
+`Some more information <https://github.com/mapserver/mapserver/wiki/Render-images-straight-out-of-S3-with-the-vsicurl-driver>`_
+
+QGIS client
+-----------
+
+Open settings, Option and fedine the following environment variable:
```
Comment (jwkaltz) [inline doc/integrator/objectstorage.rst]: "fedine" -> "define"
"variable" -> "variables"

File: doc/integrator/objectstorage.rst [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+
+* ``AWS_ACCESS_KEY_ID``: The project access key.
+* ``AWS_SECRET_ACCESS_KEY``: The project secret key.
+* ``AWS_DEFAULT_REGION=ch-dk-2``: The region use by Exoscale.
```
Comment (jwkaltz) [inline doc/integrator/objectstorage.rst]: used

File: doc/integrator/objectstorage.rst [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+Then you can add a raster layer with:
+
+* Open Data Source Manager,
+* Raster,
+* Protocol: HTTP(S), cloud, etc.,
+* Type: AWS S3
+* Bucket or container: <bucket>
+* Object key: <folder>/index.vrt
+
+We do nearly the same for vector layer.
```
Comment (jwkaltz) [inline doc/integrator/objectstorage.rst]: What does "nearly" mean? Is it the same procedure, or does something need to be done differently?

Comment (sbrunner) [reply doc/integrator/objectstorage.rst]: The only difference is that we should use the vector menu instance of the raster menu...

For me it's clear if we have the QGIS interface in front of the eyes, but I open to a better formulation :-)

**Comment by sbrunner** [comment]
Thanks for your comments :-)

File: doc/integrator/objectstorage.rst [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+   CONFIG "CPL_VSIL_CURL_CACHE_SIZE" "128000000"
+   CONFIG "CPL_VSIL_CURL_USE_HEAD" "FALSE"
+   CONFIG "GDAL_DISABLE_READDIR_ON_OPEN" "TRUE"
+
+   CONFIG "AWS_ACCESS_KEY_ID" "${AWS_ACCESS_KEY_ID}"
+   CONFIG "AWS_SECRET_ACCESS_KEY" "${AWS_SECRET_ACCESS_KEY}"
+   CONFIG "AWS_DEFAULT_REGION" "${AWS_DEFAULT_REGION}"
+   CONFIG "AWS_S3_ENDPOINT" "${AWS_S3_ENDPOINT}"
+
+Use the vrt in the layer:
```
Comment (pvalsecc) [inline doc/integrator/objectstorage.rst]: It's not a vrt, but a tile index.

File: doc/integrator/objectstorage.rst (hunk 8 above)
Comment (sbrunner) [reply doc/integrator/objectstorage.rst]: Thanks :-)

File: doc/integrator/objectstorage.rst (hunk 9 above)
Comment (sbrunner) [reply doc/integrator/objectstorage.rst]: Effectively thanks :-)

**Pull request merged**